etc/systemd/system-generators/zfs-mount-generator.in

```diff
@@ -46,6 +46,8 @@ fi
 # avoid regressions, this dependency is reduced to "wants" rather than
 # "requires". **THIS MAY CHANGE**
 req_dir="${dest_norm}/local-fs.target.wants/"
+zvol_reqdir="${dest_norm}/zfs-volumes.target.requires"
```

Comment:
requires will lead to a failing boot / systemd going to recovery mode - is this really what we want? why not just fail ("wants")?

Reply:
`zfs-volumes.target` will fail, but `zfs.target` only `Wants=` that target, so we're good. I'm not sure what the best policy is here---the boot should not fail, if I understand what is happening correctly.

Reply:
Ok, but needs testing

Reply:
Aah, I head some knot in my brain. This is just fine ;-)